doc/doxygen/stylesheet.css

```diff
@@ -26,3 +26,11 @@ div.doxygen-generated-exception-message {
     padding: 5px;
     font-size: 95%;
 }
+
+div.image img[src="fe_enriched_1d.png"] {
+    width:500px;
```

Comment:
Hm, bummer. What if you just saved the image in a size that is already roughly correct? Alternatively, just put a comment in front of the `@image` tag that explains where the size is set. (Which raises the question of how to put a comment into doxygen -- I don't really know...)